sdk/ml/azure-ai-ml/azure/ai/ml/entities/_builders/command.py

```diff
@@ -609,8 +610,14 @@ def _from_rest_object_to_init_params(cls, obj: dict) -> Dict:
 
         # handle limits
         if "limits" in obj and obj["limits"]:
-            rest_limits = RestCommandJobLimits.from_dict(obj["limits"])
-            obj["limits"] = CommandJobLimits()._from_rest_object(rest_limits)
+            timeout_response = obj["limits"]["timeout"]
+            # if response timeout is a binding string
+            if is_data_binding_expression(timeout_response):
+                obj["limits"] = CommandJobLimits(timeout=timeout_response)
```

Comment:
concern: what if CommandJobLimits has new fields

Reply:
At that time we can get a from_dict() function for CommandJobLimits and skip response-to-rest step

Reply:
The reason I didn't do it now is that, the response limit keys are different from CommandJobLimits attribute key, If we go the _from_dict() way, we need to manually map those keys. For example: 
```
obj["limits"]:
   "job_limit_type": "command"
   "timeout": "PT2S"
```

while for CommandJobLimits the attributes are "type" and "timeout"

Reply:
what about make it 
```python
CommandJobLimits._from_rest_dict(obj['limits'])
```

Reply:
so that the from & to logic are at the same place so easier to track

Reply:
Current logic can to simplified as
```python
obj["limits"] = CommandJobLimits._from_rest_object(obj["limits"])
```